templates/api/knowledge-panels/product/product_countries.tt.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "panel",
+  "title": "[% edq(lang('countries')) %]",
+  "elements": [
+    [% FOREACH country IN product.countries_tags %]
+    {
+      "type": "panel",
+      "title": "[% display_taxonomy_tag('countries_tags', country) %]"
+    }[% IF !loop.last %],[% END %]
```

Comment:
you can put the comma at the end of all entries, including the last one, it should be removed automatically after